renderdocui/renderdocui.csproj

```diff
@@ -627,6 +627,10 @@
   </ItemGroup>
   <ItemGroup />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>copy $(SolutionDir)scripts\android_remoteserver.bat $(TargetDir)
```

Comment:
I'll need to check if this works in my build scripts, as I run msbuild from a bash environment so 'copy' isn't available. MSBuild has built-in ways to copy files to the output directory though so I can fix it if needs be.